sycl/source/detail/device_filter.cpp

```diff
@@ -17,60 +17,76 @@ __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace detail {
 
+std::vector<std::string> tokenize(const std::string &Filter,
+                                  const std::string &Delim) {
```

Comment:
```suggestion
                                  char Delim) {
```

Reply:
I would keep `Delim` as a string to support any delimiter characters for the future usage.